io.openems.common/src/io/openems/common/utils/JsonUtils.java

```diff
@@ -719,6 +718,23 @@ public static Optional<String> getAsOptionalString(JsonElement jElement, String
 		return Optional.ofNullable(toString(toPrimitive(toSubElement(jElement, memberName))));
 	}
 
+	/**
+	 * Gets the member of the {@link JsonElement} as {@link String} if it exists,
+	 * and the alternative value otherwise.
+	 *
+	 * @param jElement    the {@link JsonElement}
+	 * @param memberName  the name of the member
+	 * @param alternative the alternative value
+	 * @return the {@link String} value or the alternative value
+	 */
+	public static String getAsStringOrElse(JsonElement jElement, String memberName, String alternative) {
+		try {
+			return getAsString(jElement, memberName);
+		} catch (OpenemsNamedException e) {
```

Comment:
Exceptions should not be used in normal control flow (e.g. https://dzone.com/articles/exceptions-as-controlflow-in-java)

I am replacing it with `return getAsOptionalString(jElement, memberName).orElse(alternative);`

Also JUnit test was missing.

Reply:
Thanks a lot, I wasn't aware of that. I'll keep it in mind.

Reply:
You're welcome. Usually this is not a big issue, but we found this to be a problem when we analyzed performance issues in the Backend Edge.Websocket using Visual.VM (https://visualvm.github.io/). During peak situations (e.g. restart of server) we had thousands of Exceptions per second which caused significant slow down.